src/renderer/shared/values/assets.js

```diff
@@ -4,5 +4,6 @@ export const GAS = '602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee796928
 
 export const ASSETS = {
   [NEO]: { name: 'NEO', symbol: 'NEO', decimals: 0 },
-  [GAS]: { name: 'GAS', symbol: 'GAS', decimals: 8 }
+  [GAS]: { name: 'GAS', symbol: 'GAS', decimals: 8 },
+  [NOS]: { name: 'nOS', symbol: 'NOS', decimals: 8 }
```

Comment:
We can't just inject `NOS` into this object because of other ways that the `ASSETS` constant is used.  For example, we when performing a `send`, we first check that all scriptHash values are NEO/GAS, or if they are NEP-5 tokens, which has implications on wether to use a ContractTransaction vs. InvocationTransaction.

Given that, we should revert this change, and instead modify the behavior in the modal itself.

Reply:
Alright, nice catch - I'll have a glance at it :P